src/ReturnTypes/Helpers/RequestExtension.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of Larastan.
+ *
+ * (c) Julio Foulquie <jfoulquie@gmail.com>
```

Comment:
Sorry, but this should contain the name of the package owner: `Nuno Maduro <enunomaduro@gmail.com>`.

Reply:
I'll change it! I wasn't sure what was the licensing policy for fully new files.

Reply:
@j3j5 Thanks!